pkg/limayaml/defaults.go

```diff
@@ -64,6 +64,7 @@ func defaultCPUType() CPUType {
 		// Since https://github.com/lima-vm/lima/pull/494, we use qemu64 cpu for better emulation of x86_64.
 		X8664:   "qemu64",
 		RISCV64: "rv64", // FIXME: what is the right choice for riscv64?
+		S390X:   "qemu", // FIXME: what is the right choice for riscv64?
```

Comment:
Comment copy-paste issue. Should be "for S390X" probably.

Reply:
Thanks, fixed